modules/openxr/action_map/openxr_action_map.cpp

```diff
@@ -503,6 +503,26 @@ void OpenXRActionMap::create_default_action_sets() {
 			"/user/vive_tracker_htcx/role/camera/output/haptic,"
 			"/user/vive_tracker_htcx/role/keyboard/output/haptic");
 	add_interaction_profile(profile);
+
+	// Create our hand interaction profile
+	profile = OpenXRInteractionProfile::new_profile("/interaction_profiles/ext/hand_interaction_ext");
+	profile->add_new_binding(default_pose, "/user/hand/left/input/aim/pose,/user/hand/right/input/aim/pose");
+	profile->add_new_binding(aim_pose, "/user/hand/left/input/aim/pose,/user/hand/right/input/aim/pose");
+	profile->add_new_binding(grip_pose, "/user/hand/left/input/grip/pose,/user/hand/right/input/grip/pose");
+	profile->add_new_binding(palm_pose, "/user/hand/left/input/palm_ext/pose,/user/hand/right/input/palm_ext/pose");
+
+	// Use pinch as primary
+	profile->add_new_binding(primary, "/user/hand/left/input/pinch_ext/value,/user/hand/right/input/pinch_ext/value");
+	profile->add_new_binding(primary_click, "/user/hand/left/input/pinch_ext/ready_ext,/user/hand/right/input/pinch_ext/ready_ext");
+
+	// Use activation as secondary
```

Comment:
The spec is unclear, what's the role of `aim_activate_ext`?

Reply:
You mean the OpenXR spec? 

> The "aim_activate" gesture is runtime defined, and it should be chosen so that the "aim" pose tracking is stable and usable for pointing at a distant target while the gesture is being performed.

That is indeed extremely vague, so the runtime defines the situation it is activated by detecting you're pointing at something. How it does that? 🤷 

So far I was under the impression this basically returns `1.0`/`true` when you extend your finger, but I could be wrong about that.

Personally this is the only element of this new profile I'm unlikely to use, until someone shows me a good use case :)